app/controllers/public-services/index.js

```diff
@@ -177,6 +203,26 @@ export default class PublicServicesIndexController extends Controller {
     this.resetPagination();
   }
 
+  @action
+  handleCreatorsFilterChange(values) {
+    this.creatorIds = values
+      .sort((a, b) => {
+        return a.fullName.localeCompare(b.fullName);
+      })
```

Comment:
Just a question, (I know other filters do a similar thing) but why are these being sorted after the selection is updated?

Reply:
It will show the selected options always sorted alphabetically. Not sure why but just kept it the same as the other filters, I guess it looks consistent/clean